src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java

```diff
@@ -30,6 +30,6 @@ public static void main(String[] args) {
     static String getPlayingTime(int playingTimeInSeconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        return playingTimeInSeconds/60/60/24 + " " + playingTimeInSeconds/60/60 + ":" + (playingTimeInSeconds/60-60) + ":" + (playingTimeInSeconds- playingTimeInSeconds/60*60) ;
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"